crates/solidity/testing/snapshots/cst_output/Block/postfix_recovery_regression/generated/0.4.11-failure.yml

```diff
@@ -33,7 +33,7 @@ Tree:
                   - (item꞉ Identifier): "a" # (101..102)
                   - (separator꞉ Period): "." # (102..103)
                   - (item꞉ Identifier): "b" # (103..104)
-              - (UNRECOGNIZED): "('\n        }')" # (104..118)
+              - (root꞉ UNRECOGNIZED): "('\n        }')" # (104..118)
```

Comment:
looks like we are missing the labels for `UNRECOGNIZED` and `MISSING` nodes. Should we add two matching labels for these?